source/common/stats/thread_local_store.h

```diff
@@ -332,7 +332,11 @@ class ThreadLocalStoreImpl : Logger::Loggable<Logger::Id::stats>, public StoreRo
     TextReadout& textReadoutFromStatNameWithTags(const StatName& name,
                                                  StatNameTagVectorOptConstRef tags) override;
     ScopePtr createScope(const std::string& name) override {
-      return parent_.createScope(symbolTable().toString(prefix_.statName()) + "." + name);
+      if (name.empty()) {
```

Comment:
are there any tests that fail if we don't make this particular change?

scope names are sanitized I think.